.editorconfig

```diff
@@ -5,7 +5,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
```

Comment:
Do we need to change indent size?

Reply:
The indent size in the tslint.json configuration (which predates the editorconfig file) is 4 spaces, so this is just correcting a discrepancy.